server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/MessageSender.java

```diff
@@ -67,4 +68,19 @@ private Mail buildMessage(MessageFactory.MetaDataWithContent message, Message jm
             throw new MessagingException("error building message to send", e);
         }
     }
+
+    private void validateUserIsInSenders(Message message, MailboxSession session) throws MailboxSendingNotAllowedException {
+        List<String> allowedSenders = ImmutableList.of(session.getUser().getUserName());
```

Comment:
why do you need to build this list ?

Reply:
It's moving code. Should I sanitize it also?

Reply:
Ok, I'll answer that once : moving code without considering change the code add no value.
It's just code move.
So I won't approve it.
If you moved code for naming things, creating abstraction et al, then the code maybe makes no sense anymore in the context of this new abstraction.
You have to crafts public APIs with care.